bin/active_wagon.rb

```diff
@@ -89,7 +89,7 @@ def environment(core: true)
       export RAILS_DB_USERNAME=hitobito
       export RAILS_DB_PASSWORD=hitobito
       export RAILS_DB_NAME=hit_#{wagon}_dev
-      export RAILS_TEST_DB_NAME=hit_#{wagon}_test
+      export RAILS_TEST_DB_NAME=hit_#{core ? "core" : wagon}_test
```

Comment:
Is it possible to also cover "dependency" wagons here? So far, it is only "youth" and "tenants", but logically, the test-DB should always be the one from the current directory to support local testing.